mirrord/layer/src/outgoing.rs

```diff
@@ -16,12 +16,15 @@ pub(crate) mod udp;
 
 /// Wrapper type for the (layer) socket address that intercepts the user's socket messages.
 #[derive(Debug)]
-pub(crate) struct MirrorAddress(pub(crate) SocketAddr);
+pub(crate) struct RemoteConnectResult {
```

Comment:
I would rename it to `RemoteConnection` or `RemoteAddresses` because suffixing it with Result makes me think it is a type alias for a specific `Result`

Reply:
great!